services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -65,13 +102,27 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         LogLevel.INFO, env=["AUTOSCALING_LOGLEVEL", "LOG_LEVEL", "LOGLEVEL"]
     )
 
+    AUTOSCALING_EC2_ACCESS: Optional[EC2Settings] = Field(auto_default_from_env=True)
+
+    AUTOSCALING_EC2_INSTANCES: Optional[EC2InstancesSettings] = Field(
+        auto_default_from_env=True
+    )
+
+    AUTOSCALING_NODES_MONITORING: Optional[NodesMonitoringSettings] = Field(
+        auto_default_from_env=True
+    )
+
+    AUTOSCALING_POLL_INTERVAL: datetime.timedelta = Field(
+        default=datetime.timedelta(seconds=10),
```

Comment:
Question: Why is this a timedelta and not expressed as a PostiveFloat?

Reply:
This allows to pass units in the ENV which are nicer to read.
see https://pydantic-docs.helpmanual.io/usage/types/#datetime-types

Reply:
So something like `P0DT00H00M5S `. Looks nice.